lightkurve/targetpixelfile.py

```diff
@@ -776,7 +776,7 @@ def _estimate_centroids_via_quadratic(self, aperture_mask):
         row_centr = Quantity(row_centr, unit='pixel')
         return col_centr, row_centr
 
-    def _aperture_photometry(self, aperture_mask, centroid_method='moments'):
+    def _aperture_photometry(self, aperture_mask, flux_method, centroid_method='moments'):
```

Comment:
For backwards compatibility, it is best to also assign a default in the helper function as follows:
```suggestion
    def _aperture_photometry(self, aperture_mask, flux_method="sum", centroid_method='moments'):
```

Reply:
Done